workspace-engine/src/contract.rs

```diff
@@ -23,9 +22,17 @@ use aurora_workspace_types::{AccountId, H256};
 use aurora_workspace_utils::{Contract, ContractId};
 #[cfg(feature = "ethabi")]
 use ethabi::{ParamType, Token};
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
```

Comment:
We already have re-exported borsh in `aurora_engine_types` crate. I think it would be better to use our version of the borsh.